NEWS.rst

```diff
@@ -80,6 +80,10 @@ The UniProt package now includes a method to search UniProt programmatically.
 The tutorial has been updated with examples of how to use the search method.
 See the chapter on Swiss-Prot and ExPASy in the tutorial for more information.
 
+Now when "flipping" compound locations with ``_flip()`` in ``Bio.SeqFeature``,
+if the strand is ``None``, the order of the sublocations is reversed, to improve
+the representation of origin-spanning locations. See issue #4611.
+
```

Comment:
Lead with taking the reverse complement of a SeqRecord, the fact the change is in a private method ``._flip()`` probably doesn't need mentioning.